src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/SimpleTypeModelBinder.cs

```diff
@@ -32,7 +45,11 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
 
             try
             {
-                var model = valueProviderResult.ConvertTo(bindingContext.ModelType);
+                var value = valueProviderResult.FirstValue;
+
+                var model = !string.IsNullOrWhiteSpace(value)
+                    ? _typeConverter.ConvertFrom(null, valueProviderResult.Culture, value)
+                    : null;
```

Comment:
would be more readable without the ternary. Also why is the first parameter null?